Add IsEnableAppAuthPopUpEnabled to Set-SPOTenant (#408)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

Accept wildcard characters: False
```

### -IsEnableAppAuthPopUpEnabled

> Enables or disables users in the tenancy to authenticate into SharePoint applications using popups. 

Reviewer: ```suggestion
Enables or disables users in the organization to authenticate SharePoint applications using popups. 
```

Reviewer 2: This parameter affects the way code in SharePoint interacts with Azure AD to get token to access APIs. In scenarios where 3rd party cookies are disabled (e.g. Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup if 3rd party cookies are disabled.

Accept wildcard characters: False
```

### -IsEnableAppAuthPopUpEnabled

Enables or disables users in the organization to authenticate SharePoint applications using popups. 

> This parameter affects the way code in SharePoint interacts with Azure AD to get tokens to access APIs. In scenarios where third party cookies are disabled (e.g. Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup in this scenario.

Reviewer: ```suggestion
This parameter affects the way code in SharePoint interacts with Azure AD to get tokens to access APIs. In scenarios where third-party cookies are disabled (such as Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup in this scenario.
```
